src/main/resources/org/jenkinsci/plugins/github/config/GitHubServerConfig/config.groovy

```diff
@@ -0,0 +1,33 @@
+package org.jenkinsci.plugins.github.config.GitHubServerConfig
+
+import org.jenkinsci.plugins.github.config.GitHubServerConfig
+
+def f = namespace(lib.FormTagLib);
+def c = namespace(lib.CredentialsTagLib)
+
+
+f.entry(title: _("Manage hooks"), field: "manageHooks") {
+    f.checkbox(default: true)
+}
+
+f.entry(title: _("Credentials"), field: "credentialsId") {
+    c.select()
+}
+
+f.optionalBlock(title: _("Custom GitHub API URL"), 
```

Comment:
remove this optional block and directly use/save apiUrl in GitHubServerConfig with default.
Less checkboxes - clear and obvious configuration.